### PR TITLE
Governance: add working group governance and working group template

### PR DESCRIPTION
**Pull request by winkyao**
Signed-off-by: winkyao <golangwink@gmail.com>

### CODE REVIEW

File: committee/wg-governance/wg-governance-zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Working Group 提供一个群组，来让社区的贡献者可以针对某个问题进行一段时间的合作。因为 Working Group 通常是解决一个跨越多个 Special Interest Group（SIG）职责范围的问题，所以它通常会由来自不同 SIG 的成员或者一些其他社区贡献者组成。Working Group 通常会在问题被解决之后解散，在 Working Group 中创建的成果，或者成员，可以后续被 SIG 接收。更详细的创建或者解散程序，可以参见 [WG Lifecycle](#wg-lifecycle)
+
+Working Group 具有以下特征：
+
+* 不拥有对代码的所有权
+* 有明确的可以衡量的目标，以及可以被交付的结果
+* 是一个短期的组织，并且会在目标或者使命被完成以后被解散
+
+## Working Group 与 Special Interest Group 的关系
```
Comment (Soline324) [inline committee/wg-governance/wg-governance-zh_CN.md]: Shall we add [this picture](https://download.pingcap.com/images/blog-cn/tidb-community-upgrade/5.png) here for a better idea of the relationship between Working Group and Special Interest Group?

File: committee/wg-governance/wg-governance-zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Working Group 的页面上需要在 `Stakeholder SIGs` 一节中将跟本 Working Group 相关的 SIG 列出来。
+
+## WG Lifecycle
+
+### 创建
+
+* 确定 WG 的组织者以及成员，确保成员都是 TiKV 的 Contributor。
+* 按照 [Working Group 页面模板](working-group-template.md) 在 [wg 目录](/wg) 创建一个 `xyz-wg` 的目录（将 xyz 修改为你的 WG 名称），并且将修改好的 Working Group 页面模板作为 `README.md` 页面
+* 在 [tikv/community](https://github.com/tikv/community) 提交一个 Pull Request，由 Maintainer/PMC 投票决定。再 Pull Request 被合并之后，宣布 Working Group 成立。
```
Comment (Soline324) [inline committee/wg-governance/wg-governance-zh_CN.md]:
```suggestion
* 在 [tikv/community](https://github.com/tikv/community) 提交一个 Pull Request，由 Maintainer/PMC 投票决定。在 Pull Request 被合并之后，宣布 Working Group 成立。
```

File: committee/wg-governance/working-group-template.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- Slack: [#xyz-wg](https://tikv-wg.slack.com/messages/xyz-wg)
+- [Open Community Issues/PRs](https://github.com/tikv/community/labels/wg%2Fxyz-wg)
+
+## Published Documents
+
+Put the published docs of this Working Group here.
+
+### RFC/Proposal
+
+Link your working group RFC/Proposal here.
```
Comment (Soline324) [inline committee/wg-governance/working-group-template.md]:
```suggestion
Link your Working Group RFC/Proposal here.
```

File: committee/wg-governance/working-group-template.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+## Meetings
+* Regular WG Meeting: [Mondays at 13:00 PT (Pacific Time)] (bi-weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=13:00&tz=PT%20%28Pacific%20Time%29).
+    * Link your meeting notes here.(Google Doc)
+
+## Organizers
+
+* Wink Yao (**[@winkyao](https://github.com/winkyao)**), PingCAP
+
+## Contact
```
Comment (dcalvin) [inline committee/wg-governance/working-group-template.md]:
```suggestion

## Contact
```

File: wg/README.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# Working Group
+
+A Working Group is formed by community developers who gather together to accomplish a specific goal. To achieve the goal, some WGs may span over multiple SIGs, and some WGs may only focus on something specific in a specific SIG.
+
+Each WG has its life cycle. Once the goal is completed, the group will be disbanded. The only goal of WG operations and management is to ensure that the goals set by the group are completed at the right time. In general, the working group hold periodic meetings to summarize the current project progress and determine the plans for next steps.
```
Comment (dcalvin) [inline wg/README.md]:
```suggestion
Each WG has its own life cycle. Once the goal is completed, the group will be disbanded. The only goal of WG operations and management is to ensure that the goals set by the group are completed at the right time. In general, a WG hold periodic meetings to summarize the current project progress and determine the plans for next steps.
```

File: committee/wg-governance/working-group-template.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+- Slack: [#xyz-wg](https://tikv-wg.slack.com/messages/xyz-wg)
+- [Open Community Issues/PRs](https://github.com/tikv/community/labels/wg%2Fxyz-wg)
+
+## Published Documents
+
+Put the published docs of this Working Group here.
```
Comment (dcalvin) [inline committee/wg-governance/working-group-template.md]:
```suggestion
Put the published documents of this Working Group here.
```

File: committee/wg-governance/working-group-template.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# xyz Working Group(Working Group README Template)
+
+Describe the goal of the working group, please ensure that the goal are measurable and deliverable.
+
+## Stakeholder SIGs
+
+List the SIGs who have stake holders with this Working Group.
+
+## Meetings
```
Comment (dcalvin) [inline committee/wg-governance/working-group-template.md]:
```suggestion
## Meetings

```

File: committee/wg-governance/working-group-template.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+* Regular WG Meeting: [Mondays at 13:00 PT (Pacific Time)] (bi-weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=13:00&tz=PT%20%28Pacific%20Time%29).
+    * Link your meeting notes here.(Google Doc)
```
Comment (dcalvin) [inline committee/wg-governance/working-group-template.md]:
```suggestion
    * Link your meeting notes here. (Google Doc)
```

File: committee/wg-governance/working-group-template.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
+# xyz Working Group(Working Group README Template)
```
Comment (dcalvin) [inline committee/wg-governance/working-group-template.md]:
```suggestion
# xyz Working Group (Working Group README Template)
```

File: committee/wg-governance/working-group-template.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+
+Describe the goal of the working group, please ensure that the goal are measurable and deliverable.
```
Comment (dcalvin) [inline committee/wg-governance/working-group-template.md]:
```suggestion
Describe the goal of the working group. Please ensure that the goals are measurable and deliverable.
```

File: committee/wg-governance/working-group-template.md [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+
+## Stakeholder SIGs
+
+List the SIGs who have stake holders with this Working Group.
```
Comment (dcalvin) [inline committee/wg-governance/working-group-template.md]:
```suggestion
List the stakeholder SIGs of this Working Group.
```

**Pull request merged**